src/esp/nav/PathFinder.cpp

```diff
@@ -771,8 +780,11 @@ void PathFinder::Impl::removeZeroAreaPolys() {
       CORRADE_INTERNAL_ASSERT(poly != nullptr);
       CORRADE_INTERNAL_ASSERT(tmp != nullptr);
 
-      if (polyArea(poly, tile) < 1e-5) {
+      float polygonArea = polyArea(poly, tile);
+      if (polygonArea < 1e-5) {
         navMesh_->setPolyFlags(polyRef, POLYFLAGS_DISABLED);
+      } else {
```

Comment:
```suggestion
      } else if (poly->flags & POLYFLAGS_WALK) {
```

That will make it so that is only navigable area.

Reply:
Thanks, this makes sense to accommodate future extensions. I don't see any other poly types being used in the codebase currently, is this correct or am I missing something?

Reply:
I believe recast sets the flags to all zeros by default, so anything with all zeros wouldn't be navigable, but maybe we don't have any of those?